data/setting-type.yml

```diff
@@ -54,7 +54,7 @@
   local_authority: true
   role_type: other
 
-# none -------------------------------------------------------------------------
+# none --------------------------------------- ----------------------------------
```

Comment:
There is a space here which has been added

Reply:
done